shushengfuqin/comp345_p
# Order string to log

**Summary**
This PR updates the stringToLog methods for each orders and orderlist

**Details**
_updates the stringToLog methods for each orders and orderlist
_The information of each order's execution are properly send to the gameLog.txt  
e.g.: Airlift is executed: The player has moved 3 armies from the source territory medium to the target territory large
_when add a new order to orderlist, the order type will be showed as the below:
 OrderList add order: deploy has been added.

**How to test?**
run the main.cpp ---> go to cmake-build-debug -> gameLog.txt ---> check the text are properly printed

**Checks**

- [ ] No errors when runing the main 
- [ ] The prints on gameLog.txt are correct
